Source/engine/animationinfo.cpp

```diff
@@ -174,7 +174,6 @@ void AnimationInfo::ChangeAnimationData(const CelSprite *celSprite, int numberOf
 		TickModifier = 0.0F;
 	}
 	this->pCelSprite = celSprite;
-	TicksPerFrame = ticksPerFrame;

```

Comment:
This was always a no-op, either the two values are equal or the same assignment was performed on line 171